### PR TITLE
docs(readme): Add README.md file

### LINKED ISSUE #7: Create Readme file

### LINKED ISSUE #15: Add gitter badge to README
Gitter room link: https://gitter.im/ifactory-open-source/awesome-react-app

| Style |     Code      |  Display |
|:----------|:-------------:|:------|
| Flat Square |  `[Gitter]()` | [Gitter]() |
| Plastic |    `[Gitter]()`  |   [Gitter]() |
| Normal | `[Gitter]()` |    [Gitter]() |
| Flat Blue Square | `[Gitter chat]()` | [Gitter chat]() |

we can even make our own here: http://shields.io/#your-badge

### PR DESCRIPTION
**Pull request by paulonotz0r**
This resolves #7 and resolves #15.

### CODE REVIEW

**Review by jvcjunior — APPROVED** [review]
Awesome!! Thanks a lot!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,111 @@
...
+Then open [http://localhost:3000/](http://localhost:3000/) to see your app.<br>
+When you’re ready to deploy to production, create a minified bundle with `npm run build`.
+
+
+### Get Started Immediately
+
+You **need** to have [`create-react-app`](https://github.com/facebookincubator/create-react-app) installed.
+
+You **don’t** need to install or configure tools like Webpack or Babel.<br>
+They are preconfigured and hidden so that you can focus on the code.
```
Comment (italopessoa) [inline README.md:45]: what do you think about **focus on _coding_**?

**Review by italopessoa — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (paulonotz0r) [reply README.md:45]: I think this is a very similar aproach to the existing one. Even that seems like (or it is) _different porpouses_, people will understand the same. But I agree, could be better _coding_ instead.

What do you think, @jvcjunior and @jouderianjr?

Comment (jvcjunior) [reply README.md:45]: I think we can go like that. We need the readme file ASAP. So, let's start with that and doing some refactoring over time, don't you think @italopessoa ?

Comment (italopessoa) [reply README.md:45]: Agreed @jvcjunior

**Review by italopessoa — APPROVED** [review]

**Pull request merged**